src/engine/strat_engine/blockdev.rs

```diff
@@ -225,7 +241,7 @@ impl BlockDev {
         }
     }
 
-    pub fn wipe_metadata(&mut self) -> EngineResult<()> {
+    pub fn wipe_metadata(&self) -> EngineResult<()> {
```

Comment:
block_devs is immutable because &self is immutable.  so I can't move items out of block_devs, right?